docs/developers/contributing/documentation/docs_template.md

```diff
@@ -45,6 +45,25 @@ You should include pictures/videos, particularly when describing interactions wi
 accompanied by complete and descriptive alt-text. If you're using arrows/circles to highlight portions of the image, make sure
 they contrast well against the background of the image.
 
+`````{note} Adding videos
```

Comment:
It's not letting me suggest, but the title isn't rendering. I think this is because of how notes are handled.
I think this should be:
```
{admonition} Adding videos
:class: note
```
or maybe better yet:
```
{important} Adding videos
```
Based on https://myst-parser.readthedocs.io/en/latest/syntax/roles-and-directives.html#directives-a-block-level-extension-point

Reply:
you are right, nice catch!